R/ui-styling.R

```diff
@@ -91,7 +95,8 @@ prettify_pkg <- function(transformers,
                          filetype,
                          exclude_files,
                          exclude_dirs,
-                         include_roxygen_examples) {
+                         include_roxygen_examples,
+                         pkg_root = ".") {
```

Comment:
Please also use `root` as argument name and then @inheritParams from transform_files